source/technical/networks/networks_jumbo_frames.rst

```diff
@@ -0,0 +1,66 @@
+.. _network-jumbo-frames:
+
+Jumbo Frames
+=============
+
+By default, Ethernet frames for networks created on each Chameleon site
+default to 1500 byte MTU (maximum transmission unit). However, all TOR
+switches on Chameleon are configured to allow for payloads of up to 9000 bytes.
+If you would like to experiment with jumbo frames on your private networks or
+over Layer 2 connections then please follow the steps below to implement.
+
+.. note::
+
+    You will not be able to send jumbo frames out over the public internet.
+    Neutron still defaults to only 1500 MTU so any packets that hit a DHCP
```

Comment:
I suggest you take out the mention of Neutron (I think we should try not to use too much OpenStack terminology if we can get away with it) and DHCP. DHCP doesn't actually have anything to do with the packets being dropped, in reality it is something in OVS itself, or the underlying Linux iface. I think you can just say 

> You will not be able to send jumbo frames out over the public internet, as many commercial networks do not support jumbo frames. **You also will not be able to send jumbo frames _across_ two separate tenant networks via an OpenStack router.** However, traffic between your nodes or over a stitched layer2 network like ExoGENI can all utilize jumbo frames.

The note about sending traffic across two networks is also important. Doesn't have to be in bold, that was just my emphasis.